biothings/web/query/formatter.py

```diff
@@ -174,6 +185,12 @@ def transform(self, response, **options):
             template_miss = options.pop('template_miss', dict(found=False))
             responses = [self.transform(res, **options) for res in response]
             for tpl, res in zip(templates, responses):
+                total = res.get('total', {}).get('value') or 0
+                max_total += total
+                if tpl['query'] not in count_by_queries:
+                    count_by_queries[tpl['query']] = 0
+                count_by_queries[tpl['query']] += total
+
```

Comment:
if `options.with_total` is False, we don't need to execute this block, right? @sengineer0

Reply:
No, this block still be executed.
Because if we add a check before this block, it will make the logic become more unnecessary complex, and I think the cost for calculating max_total, count_by_queries is small enough.

Reply:
@sengineer0 I don't see adding a `if` statement here adds much complexity. Also see another comment about removing `count_by_queries`

Reply:
thank for your recommendation. I am updating the code as you recommend :)
@newgene